package.json

```diff
@@ -42,7 +42,9 @@
     "karma-spec-reporter": "0.0.19",
     "mocha": "^2.2.4",
     "phantomjs": "1.9.9",
-    "wiredep": "^2.1.0"
+    "wiredep": "^2.1.0",
+    "gulp": "~3.9.1",
```

Comment:
Gulp was already included in the dependencies.
Somehow, you duplicate their declaration. Please, remove what you added. Continuous Integration (CI) used to work without this.